lib/pronto/formatter/text_formatter.rb

```diff
@@ -1,44 +1,53 @@
 module Pronto
   module Formatter
-    class TextFormatter
-      include Colorizable
+    class TextFormatter < Base
+      class TextMessageDecorator < SimpleDelegator
+        include Colorizable
+
+        LOCATION_COLOR = :cyan
+
+        LEVEL_COLORS = {
+          info: :yellow,
+          warning: :magenta,
+          error: :red,
+          fatal: :red
+        }.freeze
+
+        def to_h
+          original = __getobj__.to_h
+          original[:color_level] = format_level(__getobj__)
```

Comment:
Trying to understand pros and cons: why did you decide to leave `color_level` and `color_location` off README? 😄